data/steve_s_factory_manager/versions/A93/mod-version.cg

```diff
@@ -0,0 +1,123 @@
+schema: 1
+
+group: Required Block
+ item: Machine Inventory Manager
+     recipe:
+         input: Iron Ingot, Block of Redstone, Stone, Piston
+         pattern: 000 010 232
+         quantity: 1
+         tools: Crafting Table
+
+group: Add-on Blocks
```

Comment:
In the case of this mod, it may be better just to omit the groups.  They're completely optional, and having a "group" with a single item doesn't make much sense.  This mod is small enough where things will still make plenty of sense without them.